lib/workers/branch/lock-files.js

```diff
@@ -123,6 +123,34 @@ async function writeExistingFiles(config) {
         upath.join(basedir, 'package.json'),
         JSON.stringify(massagedFile)
       );
+
+      // if you have some reference to local files, copy them
+      const toCopy = listLocalLibs(massagedFile.dependencies);
+      toCopy.push(...listLocalLibs(massagedFile.devDependencies));
+      if (toCopy.length !== 0) {
+        logger.debug(`listOfNeededLocalFiles files to copy: ${toCopy}`);
+        await Promise.all(
+          toCopy.map(async localPath => {
+            const resolvedLocalPath = path.resolve(basedir, localPath);
+            // at the root of local Lib we should find a package.json and yarn will use it to update yarn.lock file
+            const resolvedRepoPath = upath.join(
+              resolvedLocalPath.substring(config.tmpDir.path.length + 1),
+              'package.json'
+            );
```

Comment:
Anything local here needs to be reviewed for malicious intent, or simply a bad accident. e.g. a user configuring too deep a relative path such as `../../../../` which might try to overwrite `/`.

Reply:
Good point.
I handle it now. 👍 
(+ test)